### ISSUE TITLE
Adjustments for color blindness

### ISSUE THREAD

**Issue by ascarter** [arcticicestudio/nord-atom-syntax#46]
I'm partially color blind. The theme overall is very good except for comments. They are extremely difficult to see against the background. I would be really great to have a variation that could tweak a few of the colors to provide a little more contrast for those of us that find it difficult to see.

**Comment by mattfysh** [comment]
loving this theme too! came here to also mention the comments are a little too dark to read on the dark background, especially when the active line is selected with a lighter background. any chance we could get a slightly lighter grey for comments?

**Comment by arcticicestudio** [comment]
Thanks for your proposal 👍
The low contrast is by design of Nord and has been selected specifically to not disturb code flow and keep the focus of the developer on the code. It is totally fine that you not agree to all highlighting color assignments, everyone has different preferences. Thats the reason why themes exist 😄 

Anyway, I've got a bunch of new tickets on various port projects of Nord for a increased comment contrast and even if I will not change the default color I want to make sure the theme is adaptable to fit everyone's needs.
In most cases the problem can be solved by using local overrides to adjust specific parts of the theme like I proposed in [arcticicestudio/nord-vim #26](https://github.com/arcticicestudio/nord-vim/issues/26) or [arcticicestudio/nord-emacs #23](https://github.com/arcticicestudio/nord-emacs/issues/23). Unfortunately I'm limited to this solution because the target application does not support kind of custom configurations for themes (static XML or JSON definitions).

Atom also provides a way for users to [easily apply style tweaks](https://flight-manual.atom.io/using-atom/sections/basic-customization/#style-tweaks) by using the `$ATOM_HOME/styles.less` stylesheet.]. Luckily Atom is a "hackable editor" and so I'm not limited to implement a better solution.

Long story short: I'll implement a theme setting where users can set a custom numeric value to increase the contrast / brightness of comments.

>  I would be really great to have a variation that could tweak **a few of the colors** [...]
To customize multiple colors you should then better make use of your custom `style.less` file. Syntax color variables can be found in the [`styles/syntax-variables.less`](https://github.com/arcticicestudio/nord-atom-syntax/blob/master/styles/syntax-variables.less) file. I'll also create some guides in the [project wiki](https://github.com/arcticicestudio/nord-atom-syntax/wiki) for use cases like this.

**Comment by adrian5** [comment]
I too find myself wishing for slightly brighter (~5-10%) comments when working with code where I have to read comments extensively. An adjustable theme setting sounds great! :+1:

**Comment by arcticicestudio** [comment]
I've pushed the initial implementation to the referenced PR above.
You can test it by checking out the 🌱 [feature/ghi-#46-adjustments-for-color-blindness](https://github.com/arcticicestudio/nord-atom-syntax/tree/feature/ghi-%2346-adjustments-for-color-blindness) branch 😄

**Comment by arcticicestudio** [comment]
<p align="center"><strong>Theme Settings Preview - Custom Comment Contrast</strong><br><img src="https://cloud.githubusercontent.com/assets/7836623/25037295/a8a4379c-20f8-11e7-8119-d8517cb6b76d.gif"/></p>

**Comment by arcticicestudio** [comment]
🚢 Shipped with APM package release version 🏷 [0.9.0](https://github.com/arcticicestudio/nord-atom-syntax/releases/tag/v0.9.0).
